src/createAnimatedComponent.js

```diff
@@ -344,6 +357,18 @@ export default function createAnimatedComponent(Component) {
       return props;
     }
 
+    _obtainViewName(forceComponent) {
+      const component = forceComponent || this._component;
+      try {
+        const viewConf = component.viewConfig || component.root.viewConfig;
+        return viewConf.uiViewClassName;
+      } catch (e) {
```

Comment:
why we catch here? can't we just do null checks when accessing attributes? Look like the code inside try block does not do antything other than accessing some attributes

Reply:
yes indeed I decided it's just clearer to look out for an error here than check every single level of accessed object explicitly

Reply:
If I understand this correctly you can use [optional chaining (`?.`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here

Reply:
fixed 7889b34